crates/runtime/src/memory.rs

```diff
@@ -33,6 +33,10 @@ pub trait RuntimeLinearMemory {
     /// Returns the number of allocated wasm pages.
     fn size(&self) -> u32;
 
+    /// Returns the maximum number of pages the memory can grow to.
+    /// Returns `None` if the memory is unbounded.
+    fn maximum(&self) -> Option<u32>;
```

Comment:
Could the  docs here clarify that this is about the runtime limit maximum of the memory, not the maximum size as prescribed by the type in wasm?

Reply:
I will expand these docs.